src/App.js

```diff
@@ -1,14 +1,22 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
 import Login from "./components/login/Login";
+import Dashboard from "./components/dashboard/Dashboard";
```

Comment:
rename Dashboard to index.js, then change this path to : 
`import Dashboard from "./components/dashboard";`